arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Either.kt

```diff
@@ -2062,6 +2062,7 @@ public fun <A, B> Either<A, B>.combine(SGA: Semigroup<A>, SGB: Semigroup<B>, b:
 public fun <A, B> Iterable<Either<A, B>>.combineAll(MA: Monoid<A>, MB: Monoid<B>): Either<A, B> =
   fold<Either<A, B>, Either<A, B>>(MB.empty().right()) { x, y -> Either.zipOrAccumulate(MA::combine, x, y, MB::combine) }
 
+@Deprecated(DeprecatedWiden, ReplaceWith("(this as Either<A, C>)"))
```

Comment:
```suggestion
@Deprecated(DeprecatedWiden, ReplaceWith("this"))
````

Reply:
I'm concerned if the user had a situation like:
```kotlin
val myEither: Either<Int, String> = TODO()
val widened = myEither.widen<_, _, CharSequence>()
widened.foo() // behaviour would change here
fun String.foo() {}
fun CharSequence.foo() {}
```